pkg/nodebootstrap/userdata.go

```diff
@@ -51,20 +46,19 @@ func addFilesAndScripts(config *cloudconfig.CloudConfig, files configFiles, scri
 				Path: dir + fileName,
 			}
 			if file.isAsset {
-				data, info, err := getAsset(fileName)
+				data, err := getAsset(fileName)
 				if err != nil {
 					return err
 				}
 				f.Content = data
-				f.Permissions = fmt.Sprintf("%04o", uint(info.Mode()))
```

Comment:
We didn't actually use it for anything, as it posed a challenge on the build, I figured there is no point of keeping this (untested) feature.